docs/source/deploying-volttron/proxy_web_server.rst

```diff
@@ -0,0 +1,277 @@
+.. _Apache-Reverse-Proxy-Setup:
+
+=================
+Apache Web Server
+=================
+
+The purpose of this section is to review the necessary dependencies and configurations for setting up a
+simple reverse proxy for VOLTTRON using apache2.
+
+Configuring VOLTTRON
+^^^^^^^^^^^^^^^^^^^^
+
+For this setup, VOLTTRON should be web enabled, using the http protocol and default web port of 8080. The
+bind-web-address and volttron-central-address should be set to localhost (http://localhost:8080).
+The proxy will handle requiring https. Refer to :ref:`this section <VOLTTRON-Central-Deployment>` for guidance
+on setting up an instance of VOLTTRON Central.
+
+Install Dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+To set up a reverse proxy using apache for VOLTTRON, install apache2
+and enable the ssl, rewrite, headers, proxy, and proxy_http modules.
+This will require sudo access and a restart of the apache2 service.
+
+.. code-block::
+
+    sudo apt-get install apache2
+    sudo a2enmod ssl
+    sudo a2enmod rewrite
+    sudo a2enmod headers
+    sudo a2enmod proxy
+    sudo a2enmod proxy_http
+    sudo service apache2 restart
+
+
+Configuring Apache2
+^^^^^^^^^^^^^^^^^^^
+
+In ubuntu, the apache configuation is divided into a series of directories. For setting up a basic reverse proxy
+for VOLTTRON, there are three that we need to modify, apache2.conf, ssl.conf, and 000-default.conf.
+
+
+apache2.conf File
++++++++++++++++++
+
+The apache2.conf sits in the root apache2 directory and serves as a top level configuration file
+for the apache2 web server. The default location is at /etc/apache2/apache2.conf.
+
+An example apache2.conf config file can be found at volttron/scripts/admin/apache-proxy/apache2.conf.
+Within the file, you will need to set the User and Group options, with ServerAdmin being optional.
```

Comment:
found in the volttron repository at.

Reply:
You might include what apache2.conf is actually doing...Which is setting up the base web server with user and configuration locations. etc.  There's probably a better description of what it is doing and we should use that rather than saying its located here and our example

Reply:
Good point. Added more to this section.

Reply:
Can you change the location of apache2.conf?